app/gui/src/dashboard/components/AriaComponents/Text/Text.tsx

```diff
@@ -85,16 +87,16 @@ export const TEXT_STYLE = twv.tv({
     },
     truncate: {
       /* eslint-disable @typescript-eslint/naming-convention */
-      '1': 'block truncate ellipsis',
-      '2': 'line-clamp-2 ellipsis',
-      '3': 'line-clamp-3 ellipsis',
-      '4': 'line-clamp-4 ellipsis',
-      '5': 'line-clamp-5 ellipsis',
-      '6': 'line-clamp-6 ellipsis',
-      '7': 'line-clamp-7 ellipsis',
-      '8': 'line-clamp-8 ellipsis',
-      '9': 'line-clamp-9 ellipsis',
-      custom: 'line-clamp-[var(--line-clamp)] ellipsis',
+      '1': 'block truncate',
```

Comment:
Why did we remove `ellipsis` here?

Reply:
is `ellipsis` a valid tailwind class name?

Reply:
Oh I see, it's a shortcut for overflow-hidden, text-ellipsis nowrap